### PR TITLE
Problem: no register support for payee and counterpartyPayee

### PR DESCRIPTION
**Pull request by mmsqe**
👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

## Release Notes

- **New Features**
  - Introduced support for registering payees and counterparty payees within the relayer system.
  - Enhanced configuration for gas pricing and account management in the integration tests.
  
- **Bug Fixes**
  - Improved clarity and functionality in interchain transfer processes and fee management.

- **Documentation**
  - Updated changelog to reflect new features and enhancements.

These updates enhance the payment handling capabilities and streamline the testing environment for users.

### CODE REVIEW

File: integration_tests/ibc_utils.py [hunk 1]
```diff
@@ -217,10 +228,10 @@ def prepare_network(
             wait_for_port(port)
 
 
-def register_fee_payee(src_chain, dst_chain):
+def register_fee_payee(src_chain, dst_chain, port_id, channel_id):
```
Comment (mmsqe) [inline integration_tests/ibc_utils.py:231]:
```suggestion
def register_fee_payee(src_chain, dst_chain, port_id="transfer", channel_id="channel-0"):
```

File: x/cronos/keeper/precompiles/relayer.go [hunk 2]
```diff
@@ -91,6 +93,10 @@ func init() {
 			relayerGasRequiredByMethod[methodID] = 61781
 		case Timeout:
 			relayerGasRequiredByMethod[methodID] = 104283
+		case RegisterPayee:
+			relayerGasRequiredByMethod[methodID] = 39000
```
Comment (yihuang) [inline x/cronos/keeper/precompiles/relayer.go:97]: https://github.com/crypto-org-chain/cronos/issues/1666

File: x/cronos/keeper/precompiles/relayer.go [hunk 3]
```diff
@@ -191,8 +199,34 @@ func (bc *RelayerContract) Run(evm *vm.EVM, contract *vm.Contract, readonly bool
...
+	if method.Name == RegisterPayee || method.Name == RegisterCounterpartyPayee {
+		execErr := stateDB.ExecuteNativeAction(precompileAddr, converter, func(ctx sdk.Context) error {
+			portID := args[0].(string)
+			channelID := args[1].(string)
+			caller := sdk.AccAddress(contract.CallerAddress.Bytes()).String()
+			if method.Name == RegisterPayee {
+				payeeAddr := sdk.AccAddress(args[2].(common.Address).Bytes()).String()
+				_, err := bc.ibcFeeKeeper.RegisterPayee(
+					ctx,
+					ibcfeetypes.NewMsgRegisterPayee(portID, channelID, caller, payeeAddr),
```
Comment (yihuang) [inline x/cronos/keeper/precompiles/relayer.go:212]: what about handle it similarly to the other relayer msgs, would that be simpler?

File: x/cronos/keeper/precompiles/relayer.go (hunk 2 above)
Comment (mmsqe) [reply x/cronos/keeper/precompiles/relayer.go:97]: I can try approach to the cost of cosmos tx, but these would also be static number right?

Comment (yihuang) [reply x/cronos/keeper/precompiles/relayer.go:97]: maybe no better way, it's already in a `init` function.

**Review by yihuang — APPROVED** [review]

**Pull request merged**